windows/output/core/10.0.14393.1715/tfs/2017-u3/minimal/vs2017/dockerfile

```diff
@@ -0,0 +1,19 @@
+# escape=`
+FROM microsoft/vsts-agent:windows-core-10.0.14393.1715-tfs-2017-u3
```

Comment:
Shouldn't this be microsoft/vsts-agent:windows-core-10.0.14393.1715-**2.122.1**?

Reply:
Uh, I think I got some variable names mixed around.
What do you think is the better name? with the agent version or the target name?

Reply:
I think the agent version is probably more explicit, but the target version is probably easier to reason about, so maybe the target name would be the best.

Reply:
@jmezach If I chang this I also had to change the folder structure in the output folder. I think this ist ok but it will than be different from the linux version. What do you think?